docs/appendix/upgrade-notes/2.x-to-3.0.rst

```diff
@@ -314,6 +314,7 @@ Deprecated APIs
  * ``elgg_list_entities_from_access_id``: Use ``elgg_list_entities()``
  * ``elgg_batch_delete_callback``
  * ``\Elgg\Project\Paths::sanitize``: Use ``\Elgg\Project\Paths::sanitize()``
+ * ``elgg_group_gatekeeper``: Use ``elgg_entity_gatekeeper()``
```

Comment:
what if there is no entity... i still would like to test for group membership

Reply:
ok, so i can use `entity_gate_keeper(elgg_get_pageowner_entity())` to validate if a user can access a page that is only accessible for group_members?

Reply:
Yes. Or `ElggGroup::canAccessContent()`